### ISSUE TITLE
Improve code coloring & formatting on IntelliSense info panels

### ISSUE THREAD

**Issue by adamerose** [microsoft/pylance-release#3425]
In my example below, the [quick info](https://code.visualstudio.com/docs/editor/intellisense#_intellisense-features) panel (`Ctrl + Space (twice)`) is greyscale and hard to read. I suggest the following improvements for the IntelliSense panels:

- For the quick info panel (these already exist on the mouse hover panel)
   - add syntax coloring
   - change word wrapping behavior so words don't get broken up across multiple lines
- For both the quick info panel and mouse hover panel (see the PyCharm example showing these 2 features)
  - apply code autoformatting for newlines and indentation
  - add inferred type hints for arguments and return values

![image](https://user-images.githubusercontent.com/10884874/193376850-266be159-01bd-4ac5-82c1-6498e605b840.png)

**Comment by rchiodo** [comment]
Was there a decision on this item? Hover is shown by VS code, so we'd only be able to add/remove text (not change colors)

**Comment by judej** [comment]
This is probably a dupe. If not, this needs to go to discussions.

**Comment by bschnurr** [comment]
formatting updated.  We dont control wordwrapping.. There is an inlay hint setting to see inferred return values.

This issue has been fixed in prerelease version 2023.2.11, which we've just released. You can find the changelog here: [CHANGELOG.md](https://github.com/microsoft/pylance-release/blob/main/CHANGELOG.md#2023211-1-february-2023-prerelease)

**Comment by adamerose** [comment]
> We dont control wordwrapping

Is that an intrinsic limitation of the LSP, or where should I create an issue to address this? Having the function arguments entirely cut off without any line breaks or even a horizontal scrollbar makes using VS Code for Python much harder

![image](https://user-images.githubusercontent.com/10884874/217356070-92ba4ede-74f8-40a9-bff0-aea091691c37.png)

> This issue has been fixed in prerelease version 2023.2.11

Which part was fixed? In the latest version it seems like the function arguments preview in the Quick Info panel has disappeared completely. 

<img width="478" alt="image" src="https://user-images.githubusercontent.com/10884874/217354998-9aaeefb8-c3f0-4399-8b22-407911694efc.png">

The color coding in the Hover Panel is different but still doesn't differentiate between argument name and type (the new coloring is less readable than what I showed in my original screenshot in my opinion). My suggestion was to take the old Hover Panel styling and apply it to the Quick Info panel.

<img width="804" alt="image" src="https://user-images.githubusercontent.com/10884874/217354456-d4e741d3-9551-41c7-bf59-3a882e884d8d.png">

All the problems in bullet points in my original issue description still seem to exist or have regressed.
